renovate.json

```diff
@@ -4,7 +4,7 @@
     "config:base",
     ":separateMultipleMajorReleases"
   ],
-  "prConcurrentLimit": 5,
+  "prConcurrentLimit": 4,
```

Comment:
Reduced the number or concurrent pull request opened by renovate now that we could clean-up some `on-hold`.